cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -298,6 +303,12 @@ runQueryTip (AnyConsensusModeParams cModeParams) network mOutFile = do
 -- via the local state query protocol.
 --
 
+minNtcVersionForQueryUTxOFilter :: QueryUTxOFilter -> NodeToClientVersion
```

Comment:
Should probably add a comment like "See `data NodeToClientVersion` in `ouroboros-network` for more details". There also appears to be more version related information by `data QueryVersion`. We should aim to have all queries in the cli accounted for here so when we have a new query the type checker complains we have a missing pattern match.

Reply:
I'm not sure what happened, but `minNtcVersionForQueryUTxOFilter` doesn't exist anymore.